public/_templates/bulma/css/forum.css

```
@@ -109,3 +109,11 @@ a.neutral-link {
     hyphens: auto;
   }
 }
+
+/* More wide searchbar */
```

Just a small wording suggestion:

```suggestion
/* Widen search bar to fill available space */
```

---

Thanks for your suggestion! Fix coming soon...